adapters/InMobi/InMobiAdapter/GADMAdapterInMobiConstants.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC.
+// Copyright 2022 Google LLC
```

Comment:
Revert back to `2019` since this file was created that time.

Reply:
done